src/common/JSONImporter.js

```diff
@@ -27,7 +27,25 @@ define([
             );
         }
 
-        async toJSON (node, shallow=false) {
+        async toJSON(node, omit=new Set()) {
+            if(!omit) {
```

Comment:
I wouldn't worry about this case (ie, when `omit` is explicitly set to `null`) in favor of a simpler code. If we want stricter enforcement of the correct arguments passed, we should use typescript rather than adding exhaustive runtime checking (hurting perf and code complexity).

Reply:
Changed.